nautobot/ipam/models.py

```diff
@@ -305,13 +305,24 @@ def family(self):
             return self.prefix.version
         return None
 
-    def get_utilization(self):
+    def get_percent_utilized(self):
+        """Gets the percentage utilized from the get_utilization method.
+
+        Returns
+            float: Percentage utilization
         """
-        Determine the prefix utilization of the aggregate and return it as a percentage.
+        utilization = self.get_utilization()
+        return int(utilization.numerator / float(utilization.denominator) * 100)
+
+    def get_utilization(self):
+        """Gets the numerator and denominator for calculating utilization of an Aggregrate.
+
+        Returns:
+            namedtuple: Representing the Aggregrate utilization
```

Comment:
Seems this should say it's returning a `UtilizationData` named tuple as well, in keeping w/ the others?